plugins/func/artifacthub-pkg.yml

```diff
@@ -0,0 +1,41 @@
+# Copyright 2023 The Knative Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Artifact Hub package metadata file
+# See https://github.com/artifacthub/hub/blob/master/docs/metadata/artifacthub-pkg.yml
+version: 1.9.0
+name: kn-plugin-func
```

Comment:
This name is just ID, but might be `func` only.

Reply:
Evergreen question 🌲 
I prefer `func` but am OK either way